src/ThemeContext/ThemeContext.js

```diff
@@ -1,2 +1,55 @@
-import {createContext} from "react";
-export const ThemeContext = createContext({});
+import React, {
+    createContext, useEffect, useState
+} from "react";
+
+import PropTypes from "prop-types";
+
+import LOCAL_STORAGE_KEYS from "../Viewer/services/LOCAL_STORAGE_KEYS";
+import {THEME_NAMES} from "./constants";
+
+
+const DEFAULT_THEME_NAME = THEME_NAMES.DARK;
+
+const ThemeContext = createContext(DEFAULT_THEME_NAME);
+
+/**
+ * Provides a theme context for its children components.
```

Comment:
```suggestion
 * Provides a theme context for its child components.
```

Reply:
Might be good to describe the difference between a regular React context and this context.

Reply:
Something like this?
> Provides a theme context for its child components **and manages the theme state.**

Reply:
Yep